source/ed/ed_reader.cpp

```diff
@@ -1425,10 +1425,11 @@ EdReader::get_duration (nt::Mode_e mode, float len, uint64_t source, uint64_t ta
     }
 }
 
-void EdReader::fill_graph(navitia::type::Data& data, pqxx::work& work) {
+void EdReader::fill_graph(navitia::type::Data& data, pqxx::work& work, bool export_georef_edges_geometries) {
     std::string request = "select e.source_node_id, target_node_id, e.way_id, "
                           "ST_LENGTH(the_geog) AS leng, e.pedestrian_allowed as pede, "
-                          "e.cycles_allowed as bike,e.cars_allowed as car from georef.edge e;";
+                          "e.cycles_allowed as bike,e.cars_allowed as car, "
+                          "ST_ASTEXT(the_geog) AS geometry from georef.edge e;";
```

Comment:
It will be nice to not generate the wkt if we do not need it, on fr-nw we loose 40sec.
If your are on something else I can make a PR on your PR